File: integrationtest/src/main/java/com/sequenceiq/it/VerboseLogReporter.java
package com.sequenceiq.it;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.testng.reporters.VerboseReporter;

class VerboseLogReporter extends VerboseReporter {
    private static final Logger LOG = LoggerFactory.getLogger(VerboseLogReporter.class);

    @Override
    protected void log(String message) {
        LOG.info(message.replaceAll("(?m)^", ""));
    }
}

File: integration-test/src/main/java/com/sequenceiq/it/VerboseLogReporter.kt
package com.sequenceiq.it

import org.slf4j.Logger
import org.slf4j.LoggerFactory
import org.testng.reporters.VerboseReporter

internal class VerboseLogReporter : VerboseReporter() {

    override fun log(message: String) {
        LOG.info(message.replace("(?m)^".toRegex(), ""))
    }

    companion object {
        private val LOG = LoggerFactory.getLogger(VerboseLogReporter::class.java)
    }
}
